### PR TITLE
Sentence casing. KB -> KiB, Fixing drawing issues

### PR DESCRIPTION
**Pull request by humdingerb**

### CODE REVIEW

**Comment by waddlesplash** [comment]
I really prefer KB over KiB. Does it have to change?

File: Source/MainWindow.cpp [hunk 1]
```diff
@@ -298,12 +313,12 @@ MainWindow::MainWindow ()
...
 	toolTip->SetHelp (helpButton, "Help (F1)");
 	toolTip->SetHelp (aboutButton, "About");
-	toolTip->SetHelp (saveButton, "Save Preset (Cmd S)");
+	toolTip->SetHelp (saveButton, "Save preset (ALT S)");
 	toolTip->SetHelp (optionsButton, "Preferences");
-	toolTip->SetHelp (previewButton, "Preview (Cmd P)");
-	toolTip->SetHelp (selectAllButton, "Select All (Cmd A)");
-	toolTip->SetHelp (deselectAllButton, "Deselect All (Cmd D)");
-	toolTip->SetHelp (smartSelectButton, "Select As Needed (Cmd M)");
+	toolTip->SetHelp (previewButton, "Preview (ALT P)");
```
Comment (waddlesplash) [inline Source/MainWindow.cpp]: ALT -> Alt, dash instead of space.

**Comment by humdingerb** [comment]
Everywhere else in Haiku there's KiB, not KB.
ALT-P, ALT+P, ALT P... a matter of taste, I guess. So, why not Alt-P...

**Comment by humdingerb** [comment]
Any ETM - estimated time of merging?

**Comment by waddlesplash** [comment]
Thanks -- sorry for the delay!

**Pull request merged**